app/interactors/get_last_monthly_stock_link.rb

```diff
@@ -4,52 +4,61 @@
 class GetLastMonthlyStockLink < SireneAsAPIInteractor
   around do |interactor|
     stdout_info_log 'Visiting monthly stock distant directory'
+    fail_server_unavailable unless available?(files_domain)
+
     interactor.call
-    stdout_success_log "Retrieved last monthly stock link : #{context.link}"
+
+    stdout_success_log "Retrieved last monthly stock link name : #{context.link}"
+    fail_link_unavailable unless available?(context.link)
     puts
```

Comment:
@giglemap I didnt dare to remove the puts here, can you remind me what it was for ?